debugger/src/main/kotlin/org/rust/debugger/runconfig/RsDebugProcessConfigurationHelper.kt

```diff
@@ -0,0 +1,167 @@
+            }
+
+            LLDBRenderers.BUNDLED -> {
+                val path = PP_PATH.systemDependentAndEscaped()
+                executeConsoleCommand(threadId, frameIndex, """command script import "$path/$LLDB_LOOKUP.py" """)
+
+                // In case of cross-language projects, lldb pretty-printers should be enabled
+                // only for specific std types (but should not be enabled for arbitrary struct/enums),
+                // because `type synthetic add ... -x ".*"` overrides C++ STL pretty-printers
+                val enabledTypes = if (isCrossLanguage) RUST_STD_TYPES else listOf(".*")
+                for (type in enabledTypes) {
+                    executeConsoleCommand(threadId, frameIndex, """type synthetic add -l $LLDB_LOOKUP.synthetic_lookup -x "$type" --category Rust""")
+                    executeConsoleCommand(threadId, frameIndex, """type summary add -F $LLDB_LOOKUP.summary_lookup  -e -x -h "$type" --category Rust""")
+                }
+
+                executeConsoleCommand(threadId, frameIndex, """type category enable Rust""")
+            }
+
+            LLDBRenderers.NONE -> {
+            }
+        }
+    }
+
+    private fun GDBDriver.loadPrettyPrinters() {
+        when (settings.gdbRenderers) {
+            GDBRenderers.COMPILER -> {
+                val sysroot = checkSysroot(sysroot, "Cannot load rustc renderers") ?: return
+                val path = "$sysroot/lib/rustlib/etc".systemDependentAndEscaped()
+                // Avoid multiline Python scripts due to https://youtrack.jetbrains.com/issue/CPP-9090
+                val command = """python """ +
+                    """sys.path.insert(0, "$path"); """ +
+                    """import gdb_rust_pretty_printing; """ +
+                    """gdb_rust_pretty_printing.register_printers(gdb); """
+                executeConsoleCommand(threadId, frameIndex, command)
+            }
+
+            GDBRenderers.BUNDLED -> {
+                val path = PP_PATH.systemDependentAndEscaped()
+                val command = """python """ +
+                    """sys.path.insert(0, "$path"); """ +
+                    """import $GDB_LOOKUP; """ +
+                    """$GDB_LOOKUP.register_printers(gdb); """
+                executeConsoleCommand(threadId, frameIndex, command)
+            }
+
+            GDBRenderers.NONE -> {
+            }
+        }
+    }
+
+    private fun checkSysroot(sysroot: String?, message: String): String? {
+        if (sysroot == null) {
+            project.showBalloon(message, NotificationType.WARNING)
+        }
+        return sysroot
+    }
+
+    private fun String.systemDependentAndEscaped(): String =
+        StringUtil.escapeStringCharacters(FileUtil.toSystemDependentName(this))
+
+    companion object {
+        private val LOG: Logger = Logger.getInstance(RsDebugProcessConfigurationHelper::class.java)
+
+        private val RUST_STD_TYPES: List<String> = listOf(
```

Comment:
Let's add a comment here and in `rust_types.py` that these regexes should be synchronized